### ISSUE TITLE
Weight and Length problem giggity

### ISSUE THREAD

**Issue by LeSpou7nik** [EmpyrealHell/wolfpack-rpg-client#40]
While in a fishing tournament on mobile using the client, weight and length value of fishes all indicate 0. Points are calculated normally though.

**Comment by EmpyrealHell** [comment]
Duplicate of #31
This is a known issue, the bot doesn't reply with weight and length during tournaments. There is a fix in progress on the bot, and once that fix is complete you should start to see those values during tournaments.
